src/pages/billing/CreateGCPBillingProject.ts

```diff
@@ -8,13 +8,24 @@ import Events from 'src/libs/events'
 import { formHint, FormLabel } from 'src/libs/forms'
 import * as Utils from 'src/libs/utils'
 import { billingProjectNameValidator } from 'src/pages/billing/List'
+import { BillingAccount } from 'src/pages/billing/models/BillingAccount'
 import validate from 'validate.js'
 
 
+interface CreateGCPBillingProjectProps {
+  billingAccounts: Record<string, BillingAccount>
+  chosenBillingAccount?: BillingAccount
+  setChosenBillingAccount: (BillingAccount) => void
+  billingProjectName?: string
+  setBillingProjectName: (string) => void
+  existing: any
```

Comment:
I think `existing` is a list of strings.

Reply:
it is: I just hadn't revisited this file after the point where I figured that out